BlackJackGame/Deck.cs

```diff
@@ -48,37 +48,37 @@ public List<string> GenerateDeck()
                             break;
                     }*/
                     
-                    string card = _suites[i] + _values[j];
-                    _completeDeck.Add(card);
+                    string card = values[j] + " " +  suites[i];
+                    completeDeck.Add(card);
                 }
             }
             Console.WriteLine("\ncompleted deck:");
-            for (int i = 0; i <_completeDeck.Count; i++)
+            for (int i = 0; i <completeDeck.Count; i++)
             {
-                Console.WriteLine(_completeDeck[i]);
+                Console.WriteLine(completeDeck[i]);
             }
             //completeDeck.ForEach(Console.WriteLine);
             //Console.Write(completeDeck.Count);
             //Shuffle(completeDeck);
 
-            return _completeDeck;
+            return completeDeck;
         }
 
         public List<string> Shuffle(List<string> completeDeck)
         {
-            List<string> _shuffledDeck = completeDeck.OrderBy(x => Guid.NewGuid()).ToList();
+            List<string> shuffledDeck = completeDeck.OrderBy(x => Guid.NewGuid()).ToList();
             Console.WriteLine("\nshuffled deck:");
-            _shuffledDeck.ForEach(Console.WriteLine);
+            shuffledDeck.ForEach(Console.WriteLine);
             //DrawCard(shuffledDeck);
-            return _shuffledDeck;
+            return shuffledDeck;
         }
 
         public string DrawCard(List<string> shuffledDeck)
```

Comment:
Same deal here you, when you call this method

```
deck.DrawCard(shuffledDeck)
```

Its really weird passing a whole deck to your the object thats representing your deck of cards. if you do the thing i mentioned above, this method can be rewritten to look like 

```
deck.DrawCard()
```

if you want you can pass in the number of cards you want to draw

```
deck.DrawCard(2)
```
but thats up to you